docs/src/examples/blur-roberts/blur-roberts.jl

```diff
@@ -50,20 +47,17 @@ end
 
 PixelType, mat = img2mat(img);
 
-#=
-
-## Filters implementation
-
-The `blur!` function averages the value of each pixel with the values of all
-pixels less than `width` pixels away in manhattan distance. In order to simplify
-the implementation, the filter is applied only to pixels that are sufficiently
-far from the boundary to have all their neighbors correctly defined.
-
-Results are written in-place in a pre-allocated `dest` array. Unless otherwise
-specified, the filter is applied to the whole image, but can be reduced to a
-tile if a smaller `range` argument is provided.
-
-=#
+# ## Filters implementation
+#
+# The `blur!` function averages the value of each pixel with the values of all
+# pixels less than `width` pixels away in manhattan distance. In order to
+# simplify the implementation, the filter is applied only to pixels that are
+# sufficiently far from the boundary to have all their neighbors correctly
+# defined.
+#
+# Results are written in-place in a pre-allocated `dest` array. Unless otherwise
+# specified, the filter is applied to the whole image, but can be reduced to a
+# tile if a smaller `range` argument is provided.
```

Comment:
I guess it mostly has to do with my editor and syntax highlighting color scheme. I feel like it's easier on my eyes when the markdown parts of the file can be identified via comment markers, rather than only by a shaded color.